src/routes/resources/resources.handlers.ts

```
@@ -19,10 +19,90 @@ import type {
+      const { userId, resourceId, action } = op;
+
+      if (action === "add") {
+        // Check if upvote already exists to prevent duplicates
+        const existing = await db
+          .select()
+          .from(resourceUpvotes)
+          .where(
+            and(
+              eq(resourceUpvotes.userId, userId),
+              eq(resourceUpvotes.resourceId, resourceId)
+            )
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)